pycm/pycm_multilabel_cm.py

```diff
@@ -0,0 +1,215 @@
+# -*- coding: utf-8 -*-
+"""MultiLabelCM module."""
+from __future__ import division
+from .pycm_error import pycmVectorError, pycmMultiLabelError
+from .pycm_param import *
+from .pycm_obj import ConfusionMatrix
+import numpy
+
+
+class MultiLabelCM():
+    """
+    Multilabel confusion matrix class.
+
+    >>> mlcm = MultiLabelCM([{'dog'}, {'cat', 'dog'}], [{'cat'}, {'cat'}])
+    >>> mlcm.actual_vector_multihot
+    [[0, 1], [1, 1]]
+    >>> mlcm.predict_vector_multihot
+    [[1, 0], [1, 0]]
+    """
+
+    def __init__(
+            self,
+            actual_vector,
+            predict_vector,
+            sample_weight=None,
+            classes=None):
+        """
+        Init method.
+
+        :param actual_vector: actual vector
+        :type actual_vector: python list or numpy array of list or set
+        :param predict_vector: vector of predictions
+        :type predict_vector: python list or numpy array of list or set
+        :param sample_weight: sample weights list
+        :type sample_weight: list
+        :param classes: ordered labels of classes
+        :type classes: list
+        """
+        self.actual_vector = actual_vector
+        self.actual_vector_multihot = []
+        self.predict_vector = predict_vector
+        self.predict_vector_multihot = []
+        self.weights = None
+        self.classes = None
+        self.classwise_cms = {}
+        self.samplewise_cms = {}
+        __mlcm_vector_handler__(
+            self,
+            actual_vector,
+            predict_vector,
+            sample_weight,
+            classes)
+        __mlcm_assign_classes__(self, classes)
+        __mlcm_vectors_filter__(self)
+
+    def classwise_cm(self, class_name):
```

Comment:
After testing this implementation, I want to say that this name is a bit confusable!! Because `classwise_cm` and `samplewise_cm` are methods, ' classwise_cms` and `samplewise_cms` are attributes.

My suggestion:

Changing methods name to (`get_cm_by_sample`, `get_cm_by_class`) or (`get_classwise_cm`, `get_samplewise_cm`)

Reply:
You are right. But suggested names are a bit long.

Reply:
I think `get_cm_by_sample` and `get_cm_by_class` are good.

Reply:
Done